http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1393,6 +1451,95 @@ components:
         customClusterEnvironmentVariables:
           type: object
           description: Optional environment variables to be set on the cluster.
+    CreateRuntimeRequest:
+      description: ""
+      type: object
+      properties:
+        labels:
+          type: object
+          description: The labels to be placed on the cluster. Of type Map[String,String]
+        userJupyterExtensionConfig:
+          $ref: "#/components/schemas/UserJupyterExtensionConfig"
+        jupyterExtensionUri:
+          type: string
+          description: Deprecated. Please use userJupyterExtensionConfig.
+        jupyterUserScriptUri:
+          type: string
+          description: >
+            Optional GCS object URI to a bash script the user wishes to run
+            inside their jupyter
```

Comment:
Seems odd this is double spaced.. did the formatter do that?

Reply:
weird. might be vsstudio